sources/model/managars/GGTrackerManager.m

```diff
@@ -1494,7 +1495,22 @@ - (void)reviveWatchedWaypoints{
         }
     }];
 }
-
+- (void)sendCustomerConnectedEventWithCompletionHandler:(nullable SocketResponseBlock)completionHandler {
+    if (_customerToken!=nil && [_appCustomer customerId]!=0 ) {
+        [self.liveMonitor sendCustomerSuccessEventWithCustomerAccessToken:_customerToken
+                                                               customerId:[@(_appCustomer.customerId) stringValue]
+                                                        completionHandler:completionHandler ];
+    }
+    else {
```

Comment:
i don't think this should be an error as we don't force them to pass a customer but we do want to pass the dev token.
i think it should be - use the customer details if exists otherwise just send the dev token